src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -2622,6 +2622,7 @@ public static void Format_Invalid()
             Assert.Throws<FormatException>(() => string.Format(formatter, "{-1}", obj1, obj2, obj3, obj4));
 
             // Format has out of range value
+#pragma warning disable IDE0043 // Invalid format string
```

Comment:
Strictly speaking, not really related to this particular PR... but VS seem to warn about it, although it's okay since that's the whole point of this test.